### PR TITLE
added the digital Technologies Sensors

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/digitalTechnologies/humidity/Sensor - Humidity.svg [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<svg width="500" height="500" viewBox="0 0 500 500" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (mariusgiger) [inline types/digitalTechnologies/humidity/Sensor - Humidity.svg:1]: use filenames without spaces

File: types/digitalTechnologies/humidity/battery_status.schema.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (mariusgiger) [inline types/digitalTechnologies/humidity/battery_status.schema.json:1]: normally we call this topic lifecycle

File: types/digitalTechnologies/humidity/meta.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Humidity Sensor",
```
Comment (mariusgiger) [inline types/digitalTechnologies/humidity/meta.json:2]: is this the name we want to use or Wireless Humidity Sensor?

File: types/digitalTechnologies/humidity/uplink.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  if (eventType === "humidity") {
+    sample.temperature = event.data.event.data.humidity.temperature;
+    sample.humidity = event.data.event.data.humidity.relativeHumidity;
+  } else if (eventType === "touch") {
+    sample.touch = true;
+  } else if (eventType === "networkStatus") {
+    sample.signalStrength = event.data.event.data.networkStatus.signalStrength;
+    sample.rssi = event.data.event.data.networkStatus.rssi;
+    sample.transmissionMode =
+      event.data.event.data.networkStatus.transmissionMode;
```
Comment (mariusgiger) [inline types/digitalTechnologies/humidity/uplink.js:15]: why is the input so strange? - event.data.event.data?

File: types/digitalTechnologies/humidity/uplink.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    topic = "network_status";
+  } else if (eventType === "batteryStatus") {
+    sample.batteryLevel = event.data.event.data.batteryStatus.percentage;
+    topic = "battery_status";
```
Comment (mariusgiger) [inline types/digitalTechnologies/humidity/uplink.js:19]: aren't we using lifecycle for this usually

File: types/digitalTechnologies/temperature/meta.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Temperature Sensor",
```
Comment (mariusgiger) [inline types/digitalTechnologies/temperature/meta.json:2]: Wireless?

**Review by mariusgiger — CHANGES_REQUESTED** [review]
looks good, some inputs

File: types/digitalTechnologies/humidity/uplink.js (hunk 4 above)
Comment (J2thatsme) [reply types/digitalTechnologies/humidity/uplink.js:15]: it's packaged by us into event.data and packaged by digital technologies into event.data etc

Comment (mariusgiger) [reply types/digitalTechnologies/humidity/uplink.js:15]: @giepetto ok we should look into this, I guess we should directly forward event.data without having it twice

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**